react-stopwatch-component/src/index.jsx

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+
+class StopWatch extends React.Component {
+  constructor(props) {
+    super(props);
+    this.handleClickIcon = this.handleClickIcon.bind(this);
+    this.handleClickWatch = this.handleClickWatch.bind(this);
+    this.state = { isStopped: true, time: 0, icon: 'fas fa-play' };
```

Comment:
Since you are already tracking whether the watch `isStopped`, you don't need to store the icon class in state.